python/docs/source/user_guide/pandas_on_spark/transform_apply.rst

```diff
@@ -36,7 +36,7 @@ to return the same length of the input and the latter does not require this. See
 
 In this case, each function takes a pandas Series, and pandas API on Spark computes the functions in a distributed manner as below.
 
-.. image:: https://user-images.githubusercontent.com/6477701/80076790-a1cf0680-8587-11ea-8b08-8dc694071ba0.png
+.. image:: ../../../../../docs/img/pyspark-pandas_on_spark-transform_apply01.png
```

Comment:
Can we use numbering with `1`, `2`, `3` such as
- `pyspark-pandas_on_spark-transform_apply1.png`
- `pyspark-pandas_on_spark-transform_apply2.png`
- `pyspark-pandas_on_spark-transform_apply3.png`
- ...

Instead of `01`, `02`, `03` to keep the consistency with other `png` files with numbering ??